Jenkinsfile

```diff
@@ -15,15 +10,42 @@ pipeline {
         REQUIREMENTS="${PROJECT_DIR}/bin/requirements.txt"
     }
     stages {
-        stage('Configuration scripts Tests') {
+          stage('Configuration scripts Tests') {
+              agent {
+                  docker {
+                      image 'argo.registry:5000/epel-7-py36'
+                      args '-u jenkins:jenkins'
+                  }
+              }
+              steps {
+                  echo 'Testing compute engine auto configuration scripts'
+                  sh """
+                  pip3 install -r ${REQUIREMENTS} --user
+                  pytest --junit-xml=${PROJECT_DIR}/junit.xml --cov=${PROJECT_DIR} --cov-report=xml
+                  """
+                  junit '**/junit.xml'
+                  cobertura coberturaReportFile: '**/coverage.xml'
+              }
+        }
+        stage('Flink Jobs Testing & Packaging') {
+            agent {
+                docker {
+                    image 'argo.registry:5000/epel-7-java18'
+                    args '-u jenkins:jenkins'
+                }
+            }
             steps {
-                echo 'Testing compute engine auto configuration scripts'
+                echo 'Packaging & Testing Flink Jobs'
                 sh """
-                pip3 install -r ${REQUIREMENTS} --user
-                pytest --junit-xml=${PROJECT_DIR}/junit.xml --cov=${PROJECT_DIR} --cov-report=xml
+                cd ${PROJECT_DIR}/flink_jobs/stream_status && mvn clean package cobertura:cobertura -Dcobertura.report.format=xml
```

Comment:
You could probably use the -f option in maven to specify the path to execute mvn instead of cd-ing in each directory.